magentov2/plugin/Indodana/PayLater/Helper/Transaction.php

```diff
@@ -0,0 +1,514 @@
+      'postalCode'    => $shippingAddress->getPostcode(),
+      'phone'         => $shippingAddress->getTelephone(),
+      'countryCode'   => $this->countryCode($shippingAddress->getCountryId())
+    ];
+  }
+
+  public function getSeller()
+  {
+    $sellerName = $this->_helper->getStoreName();
+    return [
+      'id'      =>  $this->_helper->getStoreID(),
+      'name'    => $sellerName,
+      'email'   => $this->_helper->getStoreEmail(),
+      'url'     => $this->_helper->getStoreUrl(),
+      'address' => [
+        'firstName'   => $sellerName,
+        'phone'       => $this->_helper->getStorePhone(),
+        'address'     => $this->_helper->getStoreAddress(),
+        'city'        => $this->_helper->getStoreCity(),
+        'postalCode'  => $this->_helper->getStorePostalCode(),
+        'countryCode' => $this->_helper->getStoreCountryCode()
+      ]
+    ];
+  }
+
+  public function getInstallmentOptions($order)
+  {
+    return $this->getIndodanaCommon()->getInstallmentOptions([
+      'totalAmount'    => $this->getTotalAmount($order),
+      'discountAmount' => $this->getTotalDiscountAmount($order),
+      'shippingAmount' => $this->getTotalShippingAmount($order),
+      'taxAmount'      => $this->getTotalTaxAmount($order),
+      'products'       => $this->getProducts($order)
+    ]);
+  }
+
+  public function checkOut($order,$paytype){
+    $approvedNotificationUrl = $this->_urlInterface->getUrl('indodanapayment/index/notify'); 
+    $cancellationRedirectUrl = $this->_urlInterface->getUrl('indodanapayment/index/cancel');
+    $backToStoreUrl = $this->_helper->getStoreUrl();
+
+    /// <!-- Development Mode
+    $approvedNotificationUrl = str_replace('localhost','192.168.1.10',$approvedNotificationUrl);
+    $cancellationRedirectUrl = str_replace('localhost','192.168.1.10',$cancellationRedirectUrl);
+    $backToStoreUrl = str_replace('localhost','192.168.1.10',$backToStoreUrl);
+    /// -->
+
+    return $this->getIndodanaCommon()->checkout(
+       [      'merchantOrderId'         => $order->getId(),
+       'totalAmount'             => $this->getTotalAmount($order),
+       'discountAmount'          => $this->getTotalDiscountAmount($order),
+       'shippingAmount'          => $this->getTotalShippingAmount($order),
+       'taxAmount'               => $this->getTotalTaxAmount($order),
+       'products'                => $this->getProducts($order),
+       'customerDetails'         => $this->getCustomerDetails($order),
+       'billingAddress'          => $this->getBillingAddress($order),
+       'shippingAddress'         => $this->getShippingAddress($order),
+       'approvedNotificationUrl' => $approvedNotificationUrl,
+       'cancellationRedirectUrl' => $cancellationRedirectUrl,
+       'backToStoreUrl'          => $backToStoreUrl 
+       ]
+     );
+  }
+
+  public function getOrderID($order){
+      return $this->_helper->getStoreID().'-'.$order->getId();
+  }
+
+  public function getOrderData($order,$paytype)
```

Comment:
I believe you don't need this code right? Because you directly use `$this->getIndodanaCommon()->checkout` on your case

If not better remove it

Reply:
ok